content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+
+A facial recognition algorithm, for example, learns to detect and recognize face edges and lines, then more significant aspects of the faces, and finally overall representations of faces. The algorithm learns and improves with time, increasing the likelihood of getting it right. In this situation, the facial recognition software will recognize faces properly over time.
+
+The ability to fine-tune the budget and the considered problem are two enhancing variables in Deep Learning. Any optimizer’s accuracy is measured using these factors.
+
+### Some Determinants That Transform Deep Learning Systems’ Performance
+
+
+ #### 1.  Taking into account the issue (considered problem) that the system will be tasked with resolving; 
+ 
+-	What kind of system should be built? 
+-	How will the developed algorithm handle the immediate user’s demands?
+-	 How easily will re-engineering be made possible?
+
+#### 2.   Evaluation difficulties of default hyperparameters for multiple optimisers.
+ Poor [accuracy](https://sigopt.com/blog/common-problems-in-hyperparameter-optimization/) is obtained here as a result of low awareness from training the system data sets. The [Adams optimiser](https://www.google.com/url?sa=t&source=web&rct=j&url=https://machinelearningmastery.com/adam-optimization-algorithm-for-deep-learning/&ved=2ahUKEwjok7uPlIL1AhWBOewKHbYnDcMQFnoECDUQAQ&usg=AOvVaw3DyIH6zRqOJ31iqVVAy1z-)  is used in this deep system algorithm, which is adaptable. 
+
+#### 3.  Technical expertise in deep learning system network design.
+The demand and need for deep learning experts is highly recommended here. Experience is needed to increase the learning rate and improve the system's performance.
+
+#### 4.  Which optimiser suits the deep network best?
+According to [research]( https://optimization.cbe.cornell.edu/index.php?title=Adam), the Adam optimiser offers the highest and best performance rate. The ability to outperform other approaches (optimisers) is lacking. 
+
+### How To Get A Perfect Performance Rate From A Deep Learning System
+>The performance rate of a deep learning system is focused on vision and pattern recognition.
+
+####  1. A Deep Neural Network System's Individual Units' (Layers') Functionality 
+
+In deep learning, [convolutional neural networks](https://reader.elsevier.com/reader/sd/pii/S1877050918308019?token=16B4CA6789714BCA2CEC6D44659321C7510FF2D8866A0E0CC6297D50E6129D2A4365D975C5F320F49E355B8662952541&originRegion=eu-west-1&originCreation=20211226214344) must take individual activation levels into account. This step, though, is difficult to achieve. First, the issue of [unit interpretability]( https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.pnas.org/content/117/48/30071&ved=2ahUKEwiSvqrstIL1AhXVOuwKHTtTDZAQFnoECAYQAQ&usg=AOvVaw2Cnz9K-g6M2q__QYAUi3h1) in deep networks must be grasped.
```

Comment:
Why are talking about CNNs in specific? Shouldn't it be generalized? Applicable for all DL models?